vision_pos_coeff: 0.001